tests/integration/templates/apigw-awsintegration-request-parameters.yaml

```diff
@@ -124,6 +144,11 @@ Resources:
         Types:
           - REGIONAL
       RegionalCertificateArn: arn:aws:acm:us-east-1:000000000000:certificate/00000000-0000-0000-0000-000000000000
```

Comment:
We'll need to fix this at some point but it's fine for now if we leave it in there :+1: